### DISCUSSION TITLE
Is latest KernelSU on non-GKI device possible?

### DISCUSSION THREAD

**Discussion by maximxlss** [tiann/KernelSU#1829] [Q&A]
Got sick of apps detecting magisk and overall wanted to try something new, so looking into installing KernelSU. Turns out my phone (joyeuse) doesn't use GKI. Seems like installing <v1.0 might work, but I want the latest version, if possible. I will build the kernel myself.

1. Is it worth trying?
2. If yes, any tips?

**Comment by mlm-games** [comment]
Have tried doing it here, you could try using it for your kernel: https://github.com/mlm-games/KernelSU-Non-GKI
